Content/ClosePrintSpooler.md

```diff
@@ -0,0 +1,28 @@
+# Close Print Spooler
+
+_ClosePrintSpooler_ is a sandbox setting that provided nuanced control over how sandboxed applications interact with the print spooler service.
+
+```
+   .
+   .
+   .
+   [DefaultBox]
+   ClosePrintSpooler=n
+```
+
+This setting can be used to prevent sandboxed applications from interacting with the print spooler service. When set to `y`, sandboxed applications will be unable to interact with the print spooler service - for example, print.
+
+Added as part of 0.5.4 / 5.46.0 version.
+
+## Interaction with [OpenPrintSpooler](OpenPrintSpooler.md)
```

Comment:
```suggestion
```

Reply:
I disagree with that change.

I specifically opted in creating a "considerable" visual separator, to indicate that the `ClosePrintSpooler` documentation has finished.

What follows is a "tangent"